phpstan.neon

```diff
@@ -15,5 +15,11 @@ parameters:
         - ../../src
     stubFiles:
         - ../../stubs/glpi_constants.php
+    ignoreErrors:
+        - '#Instantiated class PluginOcsinventoryng[A-Za-z0-9_]+ not found\.#'
+        - '#Access to property \$[A-Za-z0-9_]+ on an unknown class PluginOcsinventoryng[A-Za-z0-9_]+\.#'
+        - '#Call to method [A-Za-z0-9_]+\(\) on an unknown class PluginOcsinventoryng[A-Za-z0-9_]+\.#'
+        - '#Call to method [A-Za-z0-9_]+\(\) on an unknown class PluginFusioninventory[A-Za-z0-9_]+\.#'
+        - '#Property CommonDBTM::\$input \(array<mixed>\) does not accept false\.#'
```

Comment:
I prefer to ignore each case individually and not globally, to avoid new cases emerging.